core/backend.py

```diff
@@ -36,25 +36,25 @@ def __init__(self, model_file=PATH_TO_CKPT, label_file=PATH_TO_LABELS):
         detection_graph = tf.Graph()
         graph = tf.Graph()
         sess = tf.Session(graph=detection_graph)
-        # load the graph === 
+        # load the graph ===
         # loading a (frozen) TensorFlow mdoel into memory
-        
+
         with graph.as_default():
             od_graph_def = tf.GraphDef()
             with tf.gfile.GFile(model_file, 'rb') as fid:
                 serialized_graph = fid.read()
                 od_graph_def.ParseFromString(serialized_graph)
                 tf.import_graph_def(od_graph_def, name='')
-            
+
             # loading a label map
             label_map = label_map_util.load_labelmap(label_file)
             categories = label_map_util.convert_label_map_to_categories(label_map, max_num_classes=NUM_CLASSES, use_display_name=True)
             category_index = label_map_util.create_category_index(categories)
-        
+
         # set up instance variables
         self.graph = graph
         self.category_index = category_index
-        self.categories = categories 
+        self.categories = categories
 
     def predict(self, imageRaw):  #was origninally run_inference_for_single_image
         image = preprocess_image(imageRaw)
```

Comment:
By the way, while we're here let's remove the line below: ` print("image loaded")`